packages/flags/src/index.ts

```diff
@@ -25,6 +25,18 @@ export const isEnabled = (flag: Flag): boolean =>
   false;
 
 export const getOverrides = (): Flags => overrides;
+
+export const setManualOverrides = (flags?: Record<string, boolean>): void => {
```

Comment:
Should this just be named setOverrides to keep naming consistent?

Reply:
`setOverrides` is already taken. I renamed it as `currentOverrides`
https://github.com/yldio/asap-hub/blob/5c87fdc52cfbd92bf0449526a704fe9976e2b842/packages/react-context/src/flags.tsx#L50